conf/openerp-server.conf

```diff
@@ -0,0 +1,38 @@
+[options]
+addons_path = /usr/lib/python2.7/dist-packages/openerp/addons,/mnt/odoo-saas-tools
```

Comment:
just add /mnt/pos-addons
You don't need to copy everything to /mnt/odoo-saas-tools